starknet-crypto/src/lib.rs

```diff
@@ -1,5 +1,24 @@
+#![cfg_attr(not(feature = "std"), no_std)]
 #![doc = include_str!("../README.md")]
 
+#[cfg(feature = "std")]
+include!("./with_std.rs");
+
+#[cfg(not(feature = "std"))]
+include!("./without_std.rs");
+
+#[cfg(all(test, not(feature = "std"), feature = "alloc"))]
+include!("./with_alloc.rs");
+
+pub mod stdlib {
```

Comment:
Do we need this to be `pub`? I think we shouldn't re-export it?

Reply:
yep you are right